src/index.html

```diff
@@ -7,13 +7,15 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
     <link
       rel="stylesheet"
       href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
     />
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
-
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.3.0/js/bootstrap-datepicker.js"></script>
```

Comment:
Can we try and use this datepicker instead? https://ng-bootstrap.github.io/#/components/datepicker/overview 

It is not a good idea to mix jQuery and Angular (just like its not good to mix jQuery and React) because Angular has its own way of keeping track what is happening to the DOM.

I'm not sure why jQuery is added to this project, but we should try to stay inside the Angular ecosystem if possible.

If it is too big of a hassle, it is fine to keep the existing solution